### ISSUE TITLE
Implement a passive mode for freeroamers

### ISSUE THREAD

**Issue by TotalEpicFailure** [LVPlayground/playground#418]
This suggestion is best summed up in an example: Player_A wants to freeroam without having Player_B harass them, so he uses `/passive` (or some other word!) to grant himself invincibility and removing all weapons. Think GTA: Online here. 

There can be similar anti-abuse measures implemented akin to vehicle spawning and teleportation, but all in all I feel as though it can be great for those who just want to roam around San Andreas without fear of persistent death.

**Comment by BigShem** [comment]
Sounds like a cool idea, but what if people start abusing it, and how can we know if someone is using HH or if he is using /passive . And tbh I don't want to loose my time following someone for a long time trying to kill him then to find out that he is using passive.

**Comment by mblokker** [comment]
Simply put a 3D label above the player's head saying 'this guy is passive ok' and you will notice that in a sec. I like this idea :)

**Comment by ghost** [comment]
Sounds cool to me - it's not uncommon for players to ask for godmode for the purpose of stunting, for example, and it seems that most admins are happy to grant it.

Back in the day this would have me concerned. Take the Gangbase for example, that property was highly desirable, same with the TEF property, people/players would guard them with their life. Should we be worrying about this anymore? 

As for restrictions to activate this mode - I'm thinking 5 minutes with no weapon shots given or taken. Maybe 3 minutes.

**Comment by r2dsleep** [comment]
My suggestion is for freeroamers that they use /world command to freeroam. And they should be able to spawn vehlices in there only.

**Comment by TotalEpicFailure** [comment]
@mblokker — Great idea to tackle Shem's concerns, and something I was thinking about in my head (akin to how in GTA:O the player is transparent) but somehow didn't mention. Thank you! 

@GeorgeLVP — I don't think this is a concern, even if we _were_ talking about years ago, considering this passive mode would strip the user of their weapons. Ergo, how would they defend anything? Agreed on the restrictions, of course! We could even tie in punching like the Hide & Seek minigame (if disabling the fight key is not possible) by permitting two or three instances of punching before removing the passive mode, at which point we also institute a cooldown period before repeated usage. ☺

@r2dsleep — Virtual worlds are empty and do not lend themselves well to talking with other players in the main chat. Are there reasons you dislike this concept?

**Comment by r2dsleep** [comment]
Maybe you should just map out another world for /passive, or something. Like other servers have /stunt or /parkour.

**Comment by TotalEpicFailure** [comment]
Sleep, the whole point of this mode would be to include these players as part of the main world so that everyone gets to play together and no isolation occurs. Again, what exactly is your objection here?

**Comment by BigShem** [comment]
I think he means that LVP has alot of DMers and that we should know that it might (it will) anger more than one not to be able to chase players or to kill them in general. I think we should start a poll on the forum so we can get as much feedback as we can.

**Comment by TotalEpicFailure** [comment]
I apologize, but deathmatchers are far and away not the only players online anymore. Further, players who do not want to take part in any fighting should be permitted to do so. This is why I made a direct ticket and not a poll—freeroamers are at a massive disadvantage at the moment (especially as most fighters enjoy teleporting to unsuspecting players who at times end up quitting the server). LVP needs to be more fair toward these types of players and not only focus on the deathmatch aspect. 

We're a _playground_, remember! ;)

**Comment by BigShem** [comment]
Well, if you already know that the poll will result in a no, why add it and force it? I mean if most players don't want it to be added then just don't, I know alot of players leave after been killed, but if I started playing in LVP it was because fights were easier, and you wouldn't have players running away with super boosts like in other servers etc... That's what makes lvp unique in my opinion. We should start a poll, who knows maybe the passive mode can win.

**Comment by TotalEpicFailure** [comment]
I never said that I _knew_ a poll would fail. In fact, there's a poll now active! ☺

**Comment by mblokker** [comment]
Is there seriously a poll going on for this thing? Because there are concerns DMers don't have enough meat to pop? That's ridiculous.

**Comment by BigShem** [comment]
So for you it's okay to take decisions that might anger our players?

**Comment by TotalEpicFailure** [comment]
Shem, for whatever it may be worth, there will always be anger at any decision made. Even with ever-so-slight changes made to the game mode years and years ago, I still remember such hostility. It is an unfortunate but very real fact of not only LVP but life as a whole—no matter the decision in a conflict, there will always be an unhappy, angry, or frustrated party.

**Comment by RussellLVP** [comment]
Let me just add some colour here.

I applaud the fact that we're having a vote about this. We decided to make our development process accessible to everybody. The path towards making a decision is very much part of this. Ultimately, however, the decision itself is ours to make as stewards of the project.

There are two reasons why I'd like to see this proposal sit here for a little while before we decide anything.

Firstly, it was decided many years ago that LVP would walk the fine line between deathmatch and freeroam. Abruptly shifting too much to either side is bound to annoy people. That's understandable. That gives us reason to be careful with changes that can be impactful, like this.

However, that doesn't mean that there's not a longer-term game we can play. For example, when we introduced houses, most players preferring deathmatch disliked them. That's completely changed over the past two months. I'm sure it will change again when we enable gang zones. Such changes take time for people to understand their potential impact.

Secondly, this actually is really hard to implement correctly. We've had enough trouble trying to stop people from killing each other in interiors. This goes well beyond that - there's many more situations in which players can do each other damage in the main world. Vehicles, to name one.

Then there's the inevitable complaints we'll get. In response, we'll decide to limit it to certain areas. We'll decide to implement restrictions, and punishments if a _passive_ player kills another anyway. That's a hell of a lot of work.

Together, that makes for a potentially risky feature that will take a lot of time to implement and maintain. There are many much less risky features that will make similar impact but are much easier to implement and maintain. It seems wise to me to focus on those instead.

All of that said, yes, I agree that I would like to see something like this in the long term. I haven't yet given sufficient thought to the idea to make my mind up about an iterative way to get there. Or a friendlier, more balanced way to get there. It's a very hard issue.
